qiskit_ibm_runtime/transpiler/passes/scheduling/dynamical_decoupling.py

```diff
@@ -23,12 +23,16 @@
 from qiskit.circuit.reset import Reset
 from qiskit.dagcircuit import DAGCircuit, DAGNode, DAGInNode, DAGOpNode
 from qiskit.quantum_info.operators.predicates import matrix_equal
-from qiskit.quantum_info.synthesis import OneQubitEulerDecomposer
 from qiskit.transpiler.exceptions import TranspilerError
 from qiskit.transpiler.instruction_durations import InstructionDurations
 from qiskit.transpiler.passes.optimization import Optimize1qGates
 from qiskit.transpiler import CouplingMap
 
+try:
+    from qiskit.quantum_info.synthesis import OneQubitEulerDecomposer
+except ImportError:
+    from qiskit.synthesis import OneQubitEulerDecomposer
+
```

Comment:
Making sure this import works on both Qiskit 0.45 and 1.0 - since `OneQubitEulerDecomposer` is moved in Qiskit 1.0